lib/S3Plugin.js

```diff
@@ -334,6 +344,47 @@ class S3Plugin {
     return `${this.baseFileUrl}/${fileKey}`;
   }
 
+  /**
+   * Returns the file url
+   *
+   * @controller            file
+   * @action                getUrl
+   *
+   * @param {KuzzleRequest} request
+   * @param {Boolean} signedUrl
+   */
+  async fileGetUrl (request) {
+    this._assertCredentials()
+
+    const
+      fileKey = this._stringArg(request, 'fileKey'),
+      bucketName = this._stringArg(request, 'bucketName', this.config.bucketName),
+      bucketRegion = this._stringArg(request, 'bucketRegion', this.config.bucketRegion),
```

Comment:
You should use `request.getString(...)`instead

Reply:
this will not work in tests as we're not sending a kuzzle request but a hard coded object. How shall I update them accordingly ?

Reply:
well kuzzle-common-objects exposes Request and not KuzzleRequest so I think this comments should be taken in another issue